components/Carousel/Carousel.js

```diff
@@ -1,14 +1,54 @@
 class Carousel {
+    constructor(carousel) {
+        this.carousel = carousel;
+        console.log(this.carousel);
 
+        this.leftBtn = this.carousel.querySelector(".left-button");
+        console.log(this.leftBtn);
+
+        this.rightBtn = this.carousel.querySelector(".right-button");
+        console.log(this.rightBtn)
+
+        this.carouselImgs = this.carousel.querySelectorAll("img");
+        console.log(this.carouselImgs);
+
+        this.index = []; 
```

Comment:
Try storing your currentIndex in a simple number format starting at 0, since your users will be manually cycling through the carousel using your provided buttons